### ISSUE TITLE
ZK_REPORT_TABLE_OUTPUT_TYPE and ZK_REPORT_FORM_OUTPUT_TYPE System Configuration not work

### ISSUE THREAD

**Issue by e-Evolution** [adempiere/adempiere#1032]
the ZK_REPORT_TABLE_OUTPUT_TYPE and ZK_REPORT_FORM_OUTPUT_TYPE System Configuration not work.

Research in all code and not exist some functionality with this flags

**Comment by e-Evolution** [comment]
The problem is that the process defined the default HTML out. and this overwrite the global definition.

So the best is define any out format based in global configuration.

![captura de pantalla 2017-05-20 a la s 08 48 45](https://cloud.githubusercontent.com/assets/786968/26276480/edf062ce-3d3d-11e7-9daa-d0ad2735989f.png)

**Comment by adaxa-steven** [comment]
I am advised (paraphrased)...
"If I understand it correctly, <there is a> Global Configuration <parameter> to output the report as HTML.  However, if the Role has 'no HTML access', then it will have some issue.
Did <adaxa> cause the problem? In their context YES.  But if they allow HTML reports to restricted users, they could face the same issue we encountered".
....The issue was that the Role had insufficient control and if people could produce an html report then they were able to drill to things that their Role supposedly did not allow them to see.  They may have been restricted from opening a window by their role but the same information visible in that window could still be seen by drill-through on html reports.... or there may have been a sql select filter on (say) the BPs they could see when the BP window opened but they could get past that filter in a report.
 
So it is possible to have a conflict because the default report output type is set to html but the role is not permitted to create html reports. Better to have the possibility of that conflict than to allow data leakage.
.... and relatively unsophisticated users found the way to bypass the access control... not good!

It is possible the same issue may exist in the desktop client with normal reports but we did not check it ... and restricting users from being to create html reports in the zk client solved our immediate problem.

So Victor, correctly describes the behaviour but it is intentional .. not an error.

regards
